sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -539,6 +540,10 @@ public EditCaseSteps(
         "I click on New Sample",
         () -> webDriverHelpers.clickOnWebElementBySelector(NEW_SAMPLE_BUTTON));
 
+    When(
+        "I click on New Sample in German",
+        () -> webDriverHelpers.clickOnWebElementBySelector(NEW_SAMPLE_BUTTON_DE));
```

Comment:
we can't make a common locator for new sample  button for DE and EN?

Reply:
Not in this case. The button ID for DE version is in German (yes, the ID, not just the visible text).